CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java

```diff
@@ -79,12 +82,37 @@ public void deploy() {
 			}
 		} else {
 			deploy_build_on_linux();
+			if(!checkBuildInstallStatus()){
+				log.println("[ERROR] please check your build installation!");
+			}
 			//update CUBRID ports
 			updateCUBRIDConfigurations();
 			backup_linux();
 		}
 		
 	}
+	
+	
+	private boolean checkBuildInstallStatus(){
```

Comment:
It's better not to use checkBuildInstallStatus() here.

To be:
In script run_cubrid_install, if fail to install in any steps, just echo ERROR message.
Then we may check result whether contain ERROR to ensure it.